components/ui/navbar.js

```diff
@@ -9,12 +9,22 @@ import { useState } from "react";
 export default function Navbar() {
   const NavbarLinks = [
     {
-      path: "/speakers",
-      label: "Speakers",
-    },
+      path: "#",
+      label: "Program Guide",
+      subLinks: [
+        {
+          path: "/speakers",
```

Comment:
last Saturday we had a request to keep `schedule` and `speakers` at top level. am thinking move towards having the `dango girls` under a `community ` dropdown

Reply:
totally agree, speakers and schedule need to remain seen in the navbar

Reply:
I have restored the Schedule and Speakers but added Tutorials dropdown to link to Django Girls